content/en/platform/corda/4.9/community/blob-inspector.md

```diff
@@ -101,7 +101,7 @@ This property is materialized into `NodeInfo` and is output under the `deseriali
 
 ## Classpath
 
-If you run the blob inspector without any `.jar` files on the classpath, then it will deserialize objects using the class carpenter, (see [Object serialization](../../../../../en/platform/corda/4.8/open-source/serialization.md)).
+If you run the blob inspector without any `.jar` files on the classpath, then it will deserialize objects using the class carpenter, (see [Object serialization](serialization.md)).
```

Comment:
Same here... .html?!

Reply:
Hi Yariv, based on what @colin-hillis  said about converting in future to Antora, he thought it based to leave .md links as .md for now.